apps/content/repositories/mushaf.py

```
@@ -0,0 +1,166 @@
+        self,
+        *,
+        publisher_id: int,
+        name: str,
+        name_ar: str | None,
+        name_en: str | None,
+        description: str,
+        description_ar: str | None,
+        description_en: str | None,
+        long_description_ar: str | None,
+        long_description_en: str | None,
+        license: LicenseChoice,
```

Ruff (0.15.20)

[error] Function argument `license` is shadowing a Python builtin

(A002)